feathr-example/bureau_bureau-balance.ipynb

```diff
@@ -0,0 +1,1224 @@
+    "                          timestamp_format=\"yyyy-MM-dd HH:mm:ss\"\n",
+    "                          )\n",
+    "\n",
+    "f_MONTHS_BALANCE  = Feature(name=\"f_MONTHS_BALANCE\",\n",
+    "                        key=key_SK_ID_BUREAU, \n",
+    "                        feature_type=STRING, \n",
+    "                        transform=\"MONTHS_BALANCE\")\n",
+    "\n",
+    "f_STATUS  = Feature(name=\"f_STATUS\",\n",
+    "                        key=key_SK_ID_BUREAU, \n",
+    "                        feature_type=STRING,\n",
+    "                        transform=\"STATUS\")\n",
+    "\n",
+    "f_CREDIT_STATUS_EMA_AVG  = Feature(name=\"f_CREDIT_STATUS_EMA_AVG\",\n",
+    "                        key=key_SK_ID_BUREAU, \n",
+    "                        feature_type=STRING,\n",
+    "                        transform=\"CREDIT_STATUS_EMA_AVG\")\n",
+    "                        \n",
+    "features_bureau_balance_source_core=[\n",
+    "  f_MONTHS_BALANCE,\n",
+    "  f_STATUS,\n",
+    "  f_CREDIT_STATUS_EMA_AVG\n",
+    "  ]\n",
+    "\n",
+    "anchor_bureau_balance_source_core = FeatureAnchor(name=\"anchor_bureau_balance_source_core\",\n",
+    "                                source=bureau_balance_source_core,\n",
+    "                                features=features_bureau_balance_source_core)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# derived features BUREAU_BALANCE\n",
+    "\n",
+    "# source for aggregated features of BUREAU_BALANCE\n",
+    "# constant event_timestamp_column (1648655667 - \"epoch\")\n",
+    "bureau_balance_source_agg = HdfsSource(name=\"bureauBalanceSourceAgg\",\n",
+    "                          path=\"abfss://feathrhomecreditcafs@feathrhomecreditcasto.dfs.core.windows.net/home_credit_data/bureau_balance.csv\",\n",
+    "                          preprocessing=add_tran_date_column,\n",
+    "                          event_timestamp_column=\"TRAN_DATE\",\n",
+    "                          timestamp_format=\"yyyy-MM-dd HH:mm:ss\"\n",
+    "                          )\n",
+    "\n",
+    "f_CREDIT_STATUS_EMA_AVG_DER = DerivedFeature(name = \"f_CREDIT_STATUS_EMA_AVG_DER\",\n",
+    "                                   feature_type = STRING,\n",
+    "                                   key=[key_SK_ID_BUREAU],\n",
+    "                                   input_features = [f_CREDIT_STATUS_EMA_AVG],\n",
+    "                                   transform = \"f_CREDIT_STATUS_EMA_AVG\")\n",
+    "\n",
+    "features_bureau_balance_source_agg = [\n",
+    "  f_CREDIT_STATUS_EMA_AVG_DER\n",
+    "  ]\n",
+    "\n",
+    "anchor_bureau_balance_source_agg = FeatureAnchor(name=\"anchor_bureau_balance_source_agg\",\n",
+    "                                source=bureau_balance_source_agg,\n",
+    "                                features=features_bureau_balance_source_agg)"
```

Comment:
Correct me if I'm wrong, but I don't think we still need to define a feature anchor for a derived feature. As the derived feature is not actually anchored to a data source and they can be built directly with `client.build_features` function. This `anchor_bureau_balance_source_agg` wasn't used anywhere in the following cells also.

Reply:
Would be removing this one. This was added to test the window functionality of feathr, the `feast` version done by our client doesn't have this functionality. So make it as close as the `feast` implementation, would be removing this one.

Reply:
👍